internal/pkg/service/stream/storage/level/local/reader/readchain/chain_test.go

```diff
@@ -125,6 +125,9 @@ func TestChain_UnwrapFile_Ok(t *testing.T) {
 	file, ok = chain.UnwrapFile()
 	assert.Nil(t, file)
 	assert.False(t, ok)
+
+	// Close file
+	assert.NoError(t, expectedFile.Close())
 }
```

Comment:
In the Linux, there is no problem if a file is not closed in a test (file descriptors are automatically freed after the program ends).

But in the Windows, it fails, the `t.TempDir -> t.Cleanup` helper cannot delete the temp dir after the test.

So, I have to close all resources also in tests.